ee/fleetd-chrome/src/tables/disk_info.ts

```diff
@@ -0,0 +1,24 @@
+import Table from "./Table";
+
+export default class TableDiskInfo extends Table {
+  name = "disk_info";
+  columns = ["capacity", "id", "name", "type"];
+
+  async generate() {
+    let rows = [];
+    try {
```

Comment:
Let's remove the `try/catch`, see #12376.

TL;DR: By removing the try/catch we allow Fleet receiving the error of the live query.

Reply:
Done.